### PR TITLE
2 houses insight page add houses

### PR DESCRIPTION
**Pull request by falcaodiogo**
Insight page, static only

### CODE REVIEW

File: frontend/src/app/insight/page.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import "globals";
```
Comment (RGarrido03) [inline frontend/src/app/insight/page.tsx:1]: Remove this please, `globals.css` is already imported in `app/layout.tsx`

File: frontend/src/app/insight/page.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+export default function Home() {
+  return (
+    <div className="grid h-full flex-1 grid-cols-2">
+      <div className="grid h-full content-center rounded-tr-2xl bg-background">
+        <div className="row-auto grid content-center space-y-12 p-24">
+          <p className="text-6xl font-extrabold">
+            Welcome back, <br></br>John!
+          </p>
+          <p className="text-200 text-200 text-4xl font-medium">
```
Comment (RGarrido03) [inline frontend/src/app/insight/page.tsx:14]: `text-200` is not a Tailwind CSS classname but a Figma one, you should use `text-secondary-foreground` instead

File: frontend/src/app/insight/page.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            Here’s your summary.
+          </p>
+
+          <div className="relative rounded-xl p-4">
+            <div className="absolute inset-0 animate-pulse rounded-xl border-4 border-secondary delay-100"></div>
+            <div className="flex">
+              <div className="flex-1 pr-39 relative z-10 flex gap-x-2">
```
Comment (RGarrido03) [inline frontend/src/app/insight/page.tsx:21]: `pr-39` doesn't exist 👀 also, why such a big padding?

There are more lines with this issue, I'm just making a comment on the first

File: frontend/src/app/insight/page.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                <MaterialSymbol icon="bolt" size={32}></MaterialSymbol>
+                <p className="text-[1.2vw] font-medium">Electricity</p>
```
Comment (RGarrido03) [inline frontend/src/app/insight/page.tsx:23]: Why are you using `vw` for text size? I'm not saying it's wrong, I'm just curious on why you're not using `text-lg`, `text-xl` and etc

There are more lines with this issue, I'm just making a comment on the first

**Review by RGarrido03 — CHANGES_REQUESTED** [review]
Please check my review, thank uuu

File: frontend/src/app/insight/page.tsx (hunk 4 above)
Comment (falcaodiogo) [reply frontend/src/app/insight/page.tsx]: You're right, the size supposedly should change based on the window's width, but in this case is really not necessary.

File: frontend/src/app/insight/page.tsx (hunk 3 above)
Comment (falcaodiogo) [reply frontend/src/app/insight/page.tsx]: Ups, already deleted!!

File: frontend/src/app/insight/page.tsx (hunk 2 above)
Comment (falcaodiogo) [reply frontend/src/app/insight/page.tsx]: Ups! Thanks for the heads-up!

File: frontend/src/app/insight/page.tsx (hunk 1 above)
Comment (falcaodiogo) [reply frontend/src/app/insight/page.tsx]: Done! Won't repeat again!!

**Comment by falcaodiogo** [comment]
I've changed the code based on you're reviewing! @RGarrido03 
Thanks for the review!

**Comment by RGarrido03** [comment]
yas queen slay 💅

**Pull request merged**